mantidproject/mantid
# CryPowder:: Enables S-Empty option and Polaris system test updated

Description of work.

The S-Empty option has been enabled within the the CryPowderISIS script, this option will be triggered when a S-Empty run number is provided within the `.pref` file. S-Empty option is not being used by any of the existing system tests which is running the CryPowderISIS script, however I have added additional system test using the files provided by Helen for the Polaris instrument. 
_NOTE: The `.pref` file provided by Helen seems to be using same run number for S-Empty and V-Empty._ 

**To test:**
Code Check
If possible run `ISISPowderDiffractionXXXXTest` locally
Check that system test build pass
No further pylint violation generated



Fixes #16772 
[Release Notes](https://github.com/mantidproject/mantid/commit/5df5d12a141d4c81c77c8b6b77a062754526ab6c)



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
